src/components/Leaderboard.tsx

```diff
@@ -53,67 +68,77 @@ const Leaderboard: React.FC<IProps> = ({ content, limit = 100 }) => {
-              w="100%"
-              _hover={{
-                textDecor: "none",
-                borderRadius: 0.5,
-                boxShadow: "0 0 1px primary",
-                background: "tableBackgroundHover",
-              }}
-            >
-              <Box mr={4} opacity="0.4">
-                {idx + 1}
-              </Box>
-              <Avatar
-                src={avatarImg}
-                name={avatarAlt}
-                mr={4}
-                h={10}
-                w={10}
-                display={{ base: "none", xs: "block" }}
-              />
-              <Flex flex="1 1 75%" direction="column" mr={8}>
-                <LinkOverlay
-                  as={Link}
-                  href={hasGitHub ? `${githubUrl}${username}` : "#"}
-                  textDecor="none"
-                  color="text"
-                  hideArrow
-                >
-                  {name}{" "}
-                </LinkOverlay>
 
-                <Box fontSize="sm" opacity="0.6">
-                  {score}{" "}
-                  <Translation id="page-upgrades-bug-bounty-leaderboard-points" />
-                </Box>
-              </Flex>
-              {emoji && <Emoji mr={8} fontSize="2xl" text={emoji} />}
-              <Box
-                as="span"
-                _after={{
-                  content: '"↗"',
-                  ml: 0.5,
-                  mr: 1.5,
+          const PLACE_WORDS = [
+            "first",
+            "second",
+            "third",
+            "fourth",
+            "fifth",
+          ] as const
+          return (
+            <ListItem>
+              <LinkBox
+                key={idx}
+                display="flex"
+                justifyContent="space-between"
+                alignItems="center"
+                boxShadow={colorModeStyles.linkBoxShadow}
+                mb={0.25}
+                p={4}
+                w="100%"
+                _hover={{
+                  textDecor: "none",
+                  borderRadius: 0.5,
+                  boxShadow: "0 0 1px primary",
+                  background: "tableBackgroundHover",
                 }}
-              ></Box>
-            </LinkBox>
+              >
+                <Box mr={4} opacity="0.4">
+                  {idx + 1}
+                </Box>
+                <Avatar
+                  src={avatarImg}
+                  name={avatarAlt}
+                  mr={4}
+                  h={10}
+                  w={10}
+                  display={{ base: "none", xs: "block" }}
+                />
+                <Flex flex="1 1 75%" direction="column" mr={8}>
+                  <LinkOverlay
+                    as={Link}
+                    href={hasGitHub ? `${githubUrl}${username}` : "#"}
+                    textDecor="none"
+                    color="text"
+                    hideArrow
+                  >
+                    <VisuallyHidden>{`In ${PLACE_WORDS[idx]} place with ${score} points`}</VisuallyHidden>
```

Comment:
This is a mistake with a small const list of words to use in saying the place (first, second, etc.) as the leaderboard component can have many more than just five in the list.

Could convert to...
`In place number X with Y points`

Reply:
I like this improvement and think I prefer your second option here. I think its more worth having the second type of text than this one that is going to read `undefined` on the other links.

Another thing to have in mind is translations. This should be wrapped with the translation function & we should create a new entry for it in the `src/intl/en/common.json` file.

Reply:
How would the translation function be used here for a string that includes dynamic values?

`translationMessageId`, correct?

Reply:
That's a good question. Yes, you should use `translationMessageId` or the `Translation` component to do this.

But...since we don't have support for dynamic values in those functions yet, I would say: lets ignore the translations for this particular sentence for now.